docs/content/documentation/systems/Distributions/stochastic_tools/UniformDistribution.md

```diff
@@ -0,0 +1,18 @@
+# UniformDistribution
+!description /Distributions/UniformDistribution
```

Comment:
You can remove this, the text shown duplicates what is written below. The class description still shows up in tables that link to this class.